src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationPreambleController.cs

```diff
@@ -142,6 +143,23 @@ public IActionResult EnterApplicationUkprn(string ukprn)
             {
                 model.UKPRN = ukprn;
             }
+           
+           var ukprnTempData = TempData[_ukprn] as string;
+
+           if (!string.IsNullOrEmpty(ukprnTempData))
+           {
+               model.UKPRN = ukprnTempData;
```

Comment:
There are two ways UKPRN is being passed in the method now. I think we should stick to either passing it via model/query param or via tempdata. I prefer former as it was already being used.